docker/deploy/config/sakai.properties

```diff
@@ -1,5 +1,9 @@
 #Example configuration provided to the swarm service via the configs module of docker.
 
+# Property to define if the assignments would be sent to all the users or to the users related by the same group

```

Comment:
This property should be in the default.sakai.properties file, not this file.

It should begin by:

# S2U-30 Assignments will be assigned by default to the instructors' groups.

Reply:
Put it at the end of the assignments section:

https://github.com/sakaiproject/sakai/blob/master/config/configuration/bundles/src/bundle/org/sakaiproject/config/bundle/default.sakai.properties#L2092